src/views/layout.html

```diff
@@ -37,7 +37,9 @@
 {% endblock %}
 
 {% block bodyStart %}
-  {% include "include/cookie-consent-banner.html" %}
+  {% if FEATURE_FLAG_ENABLE_COOKIE_BANNER == "true" %}
+    {% include "include/cookie-consent-banner.html" %}
+  {% endif %}
```

Comment:
Nice, could you please add the same condition for `cookies.js` script on line 86 👍 

Please add a new task, referencing this PR and [NTRNL-439](https://technologyprogramme.atlassian.net/browse/NTRNL-439), related to the moving of static file to CDN. We do not want to have assets, added on the following [PR 53](https://github.com/cabinetoffice/github-requests-app/pull/53), thanks